Add abstract http-request patches to xod/net

evgenykochetkov opened the pull request:
We discussed that it is needed, but forgot to add it 😬

Also adds `http-get` and `http-post` that simply wrap `http-request `.

brusherru commented on workspace/__lib__/xod/net/http-request/patch.xodp:
```diff
@@ -0,0 +1,151 @@
+    },
+    {
+      "boundLiterals": {
+        "__out__": "\"\""
```
Oops, looks like default string literals aren't removed from xodball 🙈

evgenykochetkov commented:
Also, 
- renamed `xod-dev/esp8266/inet` and `xod-dev/w5500/inet` to avoid name clashing when resolving types
- renamed `xod-dev/w5500/ethernet-shield-device` to `xod-dev/w5500/w5500-device`
- added `xod-dev/w5500/ethernet-shield`
